### ISSUE TITLE
login/log in or sign in

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-backlog#306]
## What
We need an entry in the style guide with guidance on which term to use and whether or not to include a space or hyphen.

## Why
We don't have any clear guidance on what we expect people to use. We have seen questions on NHS.UK and there was a recent query and some questions at a Style Council meeting.

**Comment by sarawilcox** [comment]
The GOV.UK A to Z prefers 'sign in' to 'log in' (with a space, no hyphen): https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#sign-in-or-log-in.

There's some conversation around log in and sign in in this NHS.UK Slack thread: https://nhsuk.slack.com/archives/C0J62HNKX/p1591952619481400

**Comment by sarawilcox** [comment]
![Screenshot 2021-01-27 at 15 56 30](https://user-images.githubusercontent.com/45792959/106017730-b0b78c00-60b8-11eb-9b70-f44373f54175.png)
[Adding the NHS login button to your service/NHS login developer documentation](https://nhsconnect.github.io/nhslogin/adding-buttons/)

**Comment by simplybenuk** [comment]
Hi @sarawilcox I was wondering if there is any update on this. Reading the above and GDS feels like we should be leaning towards sign in/out. We can always change but as there is no solid recommendation, happy to find out anything we find.

**Comment by sarawilcox** [comment]
The NHS Profile editor uses "Profile editor login" as page title but "Sign in to ..." and "Log in to ..." for CTAs. https://www.nhs.uk/our-policies/profile-editor-login/

**Comment by sarawilcox** [comment]
![Screenshot 2023-08-11 at 10 13 43](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/45792959/0c9180ca-7ef5-4515-a107-6d5dc2c82d36)

https://www.nhs.uk/nhs-services/online-services/nhs-login/

**Comment by sarawilcox** [comment]
GOV.UK style guide says: 



![Screenshot 2023-08-11 at 10 16 12](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/45792959/5be83c78-ea69-4556-9160-3a4a8309d0bb)

**Comment by sarawilcox** [comment]
We also "log in" on the NHS website account page: https://www.nhs.uk/nhs-app/account/


![Screenshot 2023-08-11 at 10 18 18](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/45792959/374f773c-42f2-4c2e-ab61-aa6e72ebfaa1)

**Comment by sarawilcox** [comment]
![Screenshot 2023-08-11 at 10 21 17](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/45792959/800d0c5a-1727-40cb-8367-c909206fa69d)

When you're logged into your account on the NHS website.

**Comment by michaelgallagher** [comment]
![IMG_0677](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/771600/1d9102cd-246f-428b-a610-1fce64e27f06)

the login screen for the app

**Comment by michaelgallagher** [comment]
and the Login guidance can be found [here](https://nhsconnect.github.io/nhslogin/button-guidance/)

**Comment by sarawilcox** [comment]
From the GP registration service:
![Screenshot 2023-08-11 at 10 52 43](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/45792959/0c8c7f7a-9dfa-405b-84e8-f8fd8dff6775)

**Comment by sarawilcox** [comment]
I'll review this ticket as we work on the Header for 'logged in' users #192.

Including taking into accoun:
- [Adding the Care Identity button to your service - NHS England Digital](https://digital.nhs.uk/services/care-identity-service/applications-and-services/cis2-authentication/guidance-for-developers/adding-the-care-identity-button-to-your-service)
- [NHS login button guidance](https://nhsconnect.github.io/nhslogin/button-guidance)

**Comment by sarawilcox** [comment]
## Review of what we know about log in versus sign in

### Blog post by Tania Richardson, 2018

[Log in vs Sign in - Which is Better?](https://taniarichardson.com/log-in-vs-sign-in/) 

The internet is undecided. 

“Sign in” and “sign up” are quite close. She suggests “Register “for new users and “Login” for existing users.

“Login” for noun, “log in” for verb 

Note: single sign on (SSO)

### GOV.UK

GOV.UK style guide recommends "sign in" rather than "log in". And "Sign in with GOV.UK One Login", login” being the noun and “log in” the verb. 

GOV colleagues told me they didn’t think there were any particular UR insights behind the decision. It was more a case of picking one for consistency. GOV.UK One Login came along much later.  

Years ago, it was mentioned that "log in" tested worse with people with lower digital literacy. because there is no direct analogue comparison. People were familiar with the concept of "signing in" at a hotel, for example, but if they had low digital literacy they didn't really understand what "log in". But things have probably moved on since then. 

### Google Trends

Most people search for “login”, rather than “log in”, “sign in” or “log on” (but that may be conflating noun and verb.) 

### Consistency across NHS services

So far as we know, there are no user issues with either “sign in” or “log in”. 

The NHS generally uses log in. 

#### NHS Login

We have NHS Login (but wouldn’t say “log in to NHS Login” - see below). Users may see "NHS login" when they use the App, for example. 

Services that use NHS Login and the design system have a button that just says “Continue”.

Other services typically say: “Continue to NHS Login”. See [Adding the NHS login button to your website or app (NHS login developer documentation)](https://nhsconnect.github.io/nhslogin/button-guidance/).

![Image](https://github.com/user-attachments/assets/42cb9986-d316-4d2a-b5dd-e63c2f9815ee)

#### CIS2

The NHS Care Identity Service is staff-facing but it also uses the word "log in".

![Image](https://github.com/user-attachments/assets/aa4f1628-08ae-4ea4-ba53-93ed68f467b5)

[Guidance on adding the button (NHS Digital website)](https://digital.nhs.uk/services/care-identity-service/applications-and-services/cis2-authentication/guidance-for-developers/adding-the-care-identity-button-to-your-service)

#### NHS App

The NHS App uses "log out", as does "My account" on the NHS website. 

#### Log in component on the NHS website

The Account team developed and tested a component that appears on several NHS website pages. 

On the [repeat prescriptions page](https://www.nhs.uk/nhs-services/prescriptions/how-to-order-a-repeat-prescription/):

![Image](https://github.com/user-attachments/assets/159a074b-9c07-4658-859d-9d4e87ff2826)

On the [online health and prescriptions page](https://www.nhs.uk/nhs-services/gps/online-health-and-prescription-services/):

![Image](https://github.com/user-attachments/assets/25658411-56fd-4e7d-925f-d540512836ff)

There are still some questions around user journeys between the NHS website and the App ("My account") that may affect the use of this component but, for the time being, it was tested initially and is being used in various places on the NHS website. 

### Recommendation

That we use "log in" and "log out".

**Comment by sarawilcox** [comment]
Agreed at Style Council meeting Jan 2025 that we will add an entry to the content guide for "log in/log out".

**Comment by sarawilcox** [comment]
Published: https://service-manual.nhs.uk/content/a-to-z-of-nhs-health-writing#log-in-and-log-out.
